e2e/cypress/support/api/user.js

```diff
@@ -168,33 +168,12 @@ Cypress.Commands.add('apiCreateUser', ({prefix = 'user', bypassTutorial = true,
 Cypress.Commands.add('apiCreateGuestUser', ({prefix = 'guest', activate = true} = {}) => {
     return cy.apiCreateUser({prefix}).then(({user}) => {
         cy.demoteUser(user.id);
-        cy.apiActivateUser(user.id, activate);
+        cy.externalActivateUser(user.id, activate);
 
         return cy.wrap({guest: user});
     });
 });
 
-/**
- * Saves channel display mode preference of a user directly via API
- * This API assume that the user is logged in and has cookie to access
- * @param {String} status - "online" (default), "offline", "away" or "dnd"
- */
-Cypress.Commands.add('apiUpdateUserStatus', (status = 'online') => {
-    return cy.getCookie('MMUSERID').then((cookie) => {
-        const data = {user_id: cookie.value, status};
-
-        return cy.request({
-            headers: {'X-Requested-With': 'XMLHttpRequest'},
-            url: '/api/v4/users/me/status',
-            method: 'PUT',
-            body: data,
-        }).then((response) => {
-            expect(response.status).to.equal(200);
-            cy.wrap({status: response.body});
-        });
-    });
-});
-
```

Comment:
Wondering why this API command is moved out to a separate file. I am not sure if we should break this to separate files at this granularity. I suggest to revert back and keep user related APIs into a single user file.

Reply:
Folder structure depends on how the API reference is structured - https://api.mattermost.com/#tag/status. Looking at the link, there are 3 endpoints for it. I think that's the best way to track whether we already implemented an API endpoint or not, and to prevent potential duplicates.

Reply:
Got it. Thanks. That makes sense now :)